Frontend/ui-driver/src/Screens/HomeScreen/ScreenData.purs

```diff
@@ -175,6 +175,11 @@ initData =
       , onRideBannerTimerID :""
       , onRideBannerTimer : 0
       , scheduleRideCount : Nothing
+      , favPopUp : {
+          visibility : false,
+          title : "",
+          message : ""
+        }
```

Comment:
move to some function like `defaultFavPopUpData`. its easy to read and update the data